services/src/main/scala/cromwell/services/auth/impl/GithubAuthVendingActor.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.services.auth.impl
+
+import akka.actor.{Actor, ActorRef, Props}
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.LazyLogging
+import cromwell.core.Dispatcher.ServiceDispatcher
+import cromwell.services.auth.GithubAuthVending.{GithubAuthRequest, GithubAuthTokenResponse, NoGithubAuthResponse}
+
+class GithubAuthVendingActor(serviceConfig: Config, globalConfig: Config, serviceRegistryActor: ActorRef)
+    extends Actor
+    with LazyLogging {
+
+  lazy val enabled = serviceConfig.getBoolean("enabled")
+
+  override def receive: Receive = {
+    case GithubAuthRequest(_) if enabled =>
+      sender() ! GithubAuthTokenResponse(serviceConfig.getString("access-token"))
```

Comment:
Should we add an `access-token = ""` to `reference.conf` so that it's a complete reference? Perhaps with the comment about `Bearer` from the PR description.

Reply:
Good point. I can add this back to reference.conf with a note that it should be removed when ECM integration is added.

Reply:
Chiming in from vacation: I am expecting us to remove this hard-code option when we support calling ECM. Or at least leaving it heavily undocumented

Reply:
@cjllanwarne I have added a note in the conf file that `access-token` config should be removed when ECM integration happens but if you are inclined to remove this now then I can do that too since this is your PR. I added it back because additional to Adam's point, it made sense to me to document it and have it there for easier testing.